config/v0.3.0/resource.go

```diff
@@ -47,16 +47,19 @@ func keyVal(k string, v any) attribute.KeyValue {
 }
 
 func newResource(res *Resource) *resource.Resource {
-	if res == nil || res.Attributes == nil {
+	if res == nil {
```

Comment:
If user provide an empty `Resource` explicitly (not `nil`) then we should honor this configuration.

Reply:
Should a test be added to check this?

Reply:
Thanks, missed the second one. :+1: